assets/models/system/deepseek-r1-distill-qwen-14b/model.yaml

```diff
@@ -5,4 +5,4 @@ path:
   type: azureblob
 publish:
   description: description.md
-  type: custom_model
\ No newline at end of file
+  type: custom_model
```

Comment:
Same thing as above - appears to indicate a change where there isn't one (that's visible at least).